.nycrc

```diff
@@ -6,8 +6,10 @@
     "**/*.d.ts",
     "build/**/**/*.js",
     "src/index.ts",
+    "src/version.ts",
     "karma.conf.js",
     "src/platform/browser/*.ts",
+    "test/*.ts",
```

Comment:
why ignore the test coverage on test files?

Reply:
They are not representing code coverage. That caused my to fail duo to low code coverage.

Reply:
I understand the problem but believe we should find a proper solution to this than disable test coverage altogether on this package.
It's related to the issue of not running v3 tests in `npm run test`.
Maybe when it is resolved, this one will be too,

Reply:
never mind my comment, I read it again and understood why it works.
sorry for the trouble :)